mythtv/libs/libmythtv/dbcheck.cpp

```diff
@@ -2687,7 +2748,9 @@ static bool doUpgradeTVDatabaseSchema(void)
             "ADD COLUMN video_avg_bitrate MEDIUMINT UNSIGNED NOT NULL DEFAULT 0, " // Kbps
             "ADD COLUMN video_max_bitrate MEDIUMINT UNSIGNED NOT NULL DEFAULT 0, " // Kbps
             "ADD COLUMN audio_avg_bitrate MEDIUMINT UNSIGNED NOT NULL DEFAULT 0, " // Kbps
-            "ADD COLUMN audio_max_bitrate MEDIUMINT UNSIGNED NOT NULL DEFAULT 0 ;" // Kbps
+            "ADD COLUMN audio_max_bitrate MEDIUMINT UNSIGNED NOT NULL DEFAULT 0 ;", // Kbps
+
+            "ALTER TABLE recorded ADD COLUMN lastplay TINYINT UNSIGNED DEFAULT 0 AFTER bookmark;",
```

Comment:
Adding field lastplay should be done in the next section, the upgrade to schema 1331 because that is where the field is needed. If somebody would do an upgrade from schema version 1330 then that would still fail.....

Reply:
Done.